tests/Specs/Pack.cs

```diff
@@ -1252,5 +1253,75 @@ public void HTML_ShouldPackWithDefaultValuesIfRelated()
             steps.Elements().Count().Should().Be(4, "HTML package should have 4 installation steps");
         }
 
+
+        [Fact]
+        public void Tests()

```

Comment:
This test's name could be more specific, maybe something like:
```suggestion
        public void Pack_TestPackage()

```
?

Reply:
Done